src/Nethermind/Nethermind.Dsl/Grammar/DslGrammar.g4

```diff
@@ -1,17 +1,19 @@
 grammar DslGrammar;
 
 tree: (expression)* ;
-expression : watchExpression | whereExpression | publishExpression | andCondition | orCondition ;
+expression : sourceExpression | watchExpression | whereExpression | publishExpression | andCondition | orCondition ;
+sourceExpression : SOURCE WORD ;
 watchExpression : WATCH WORD ;
 whereExpression : WHERE condition ;
-publishExpression : PUBLISH PUBLISH_VALUE ( WORD | DIGIT ) ;
-condition : WORD BOOLEAN_OPERATOR ( WORD | DIGIT | ADDRESS | BYTECODE ) ; 
+publishExpression : PUBLISH PUBLISH_VALUE ( URL | WORD | DIGIT );
+condition : WORD BOOLEAN_OPERATOR ( WORD | DIGIT | ADDRESS | BYTECODE ) ;
 andCondition : AND condition ;
 orCondition : OR condition ;
 
 BOOLEAN_OPERATOR : ARITHMETIC_SYMBOL | CONTAINS ;
 ARITHMETIC_SYMBOL : '==' | '!=' | '<' | '>' | '<=' | '>=' | IS | NOT ;
 
+SOURCE : 'SOURCE' ;
```

Comment:
why is SOURCE added here?

Reply:
I honestly don't know, I don't remember ever writing that but I will remove it promptly